src/plugin.ts

```diff
@@ -56,6 +60,101 @@ export const tenancy =
           ? global
           : {
               ...global,
+              endpoints: [
+                {
+                  path: "/versions/",
+                  method: "get",
+                  handler: async (req: PayloadRequest, res: Response) => {
```

Comment:
Let's move these routes to a separate hanlders.ts file, you can expose a method from there createGetVersionsRoute() which will return the endpoint, same goes for other ones.